src/components/IconLibrary/IconLibrary.js

```diff
@@ -212,32 +241,64 @@ function createIconSections(icons, filteredIcons) {
       return groups[size].length !== 0;
     })
     .map(size => (
-      <section key={size} className="icon-size">
+      <section
+        key={size}
+        className="icon-size"
+        aria-labelledby={`icon-h2 icon-h2-${size}`}>
         <header>
-          <h2 className="icon-h2">{isNaN(size) ? size : `${size}x${size}`}</h2>
+          <h2 className={`icon-h2 icon-h2-${size}`}>
+            {isNaN(size) ? size : `${size}x${size}`}
+          </h2>
         </header>
-        <div className="icon-container">
-          {groups[size]
+        <ul className="icon-container" tabindex="0">
+          {renderIconList(groups[size], filteredIcons)}
+          {/* {groups[size]
             .filter(icon => filteredIcons.indexOf(icon.name) !== -1)
-            .map(renderIcon)}
-        </div>
+            .map(renderIcon)} */}
+        </ul>
       </section>
     ));
 }
 
+/**
+ * render list of available icons for this category.
+ * if none are available, return a blank block
+ */
+
+function renderIconList(categoryArray, filteredList) {
+  let displayedIconsList = [];
+  categoryArray.forEach(icon => {
+    if (filteredList.indexOf(icon.name) !== -1) {
+      displayedIconsList.push(icon);
+    }
+  });
+  const displayedIcons = displayedIconsList.map(renderIcon);
+  return displayedIcons.length > 0 ? (
```

Comment:
Might help for readability, definitely personal preference:
```suggestion
  if (displayedIcons.length > 0) {
    return displayedIconsList.map(renderIcon);
  }
  
  return (
    <li className="icon">
      <div className="icon__card" />
      <h5 className="icon__card-title" title="No results in this size.">
        No results in this size.
      </h5>
    </li>
  );
```